src/main/java/frc/robot/subsystems/IndexSubsystem.java

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.util.Color;
+
+import com.revrobotics.ColorSensorV3;
+import com.revrobotics.ColorMatchResult;
+import com.fasterxml.jackson.annotation.JsonTypeInfo.Id;
+import com.revrobotics.ColorMatch;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.motorcontrol.TalonSRXControlMode;
+import frc.robot.Constants;
+
+
+public class IndexSubsystem extends SubsystemBase {
+
+  // Should this be in constants?
+  private final I2C.Port i2cPort = I2C.Port.kOnboard;
+  int i2cPortId = 0x70;
+  public final int ColorSensorid = 0;
+
+  // Senses colors
+  private final ColorSensorV3 colorSensor = new ColorSensorV3(i2cPort);
```

Comment:
Why is this subsystem reimplementing features implemented in the `ColorSensorSubsystem`? All color sensors should be managed by the `ColorSensorSubsystem` to prevent trying to instantiate the same object on the same port/address.

Reply:
I'm using color sensors to detect if the ball is there, the color sensor subsystem was just a test to see how the sensor worked. I'll delete the color sensor subsystem + command.